plutus-core/plutus-ir/test/transform/inline/letOverAppMultiId.golden

```diff
@@ -0,0 +1,9 @@
+(let
+  (nonrec)
+  (termbind
+    (strict)
+    (vardecl idFun (fun (con integer) (con integer)))
+    (lam y (con integer) y)
+  )
+  [ [ (lam x (con integer) idFun) idFun ] (con integer 6) ]
```

Comment:
Neither of the two `idFun`s is fully saturated.

Reply:
Again, this would only work if we were running much more of the simplifier, which we shouldn't in the inliner tests anyway.

Reply:
I just realized this is the same as the one above. Deleting.